Makefile

```diff
@@ -16,7 +16,7 @@ package:
 LINT_TIMEOUT ?= 3m
 lint:
 	docker-compose -f docker-compose.test.yml build tests
-	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --deadline=$(LINT_TIMEOUT) --verbose
+	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --exclude="(conf\.(Un)?[S,s]etEnv)" --deadline=$(LINT_TIMEOUT) --verbose
```

Comment:
Can we create a ticket to fix these moving forward?

Also, can we update the settings.json so our local VS code setup can ignore these errors for now?